colossus/src/main/scala/colossus/service/ServiceDSL.scala

```diff
@@ -185,9 +151,30 @@ trait ClientLifter[C <: Protocol, T[M[_]] <: Sender[C,M]] {
 trait ClientFactory[C <: Protocol, M[_], T <: Sender[C,M], E] {
 
 
+  /**
+    * Load a ServiceClient definition from a Config.  Looks into `colossus.clients.$clientName` and falls back onto
+    * `colossus.client-defaults`
+    * @param clientName The name of the client definition to load
+    * @param config A config object which contains at the least a `colossus.clients.$clientName` and a `colossus.client-defaults`
+    * @return
+    */
+  def apply(clientName : String, config : Config = ConfigFactory.load())(implicit provider: ClientCodecProvider[C], env: E) : T = {
```

Comment:
Oh right that's not going to work here since the config is path based and you need the whole config.  Still, I think we can probably do a `Config.load`once in the `ServiceClient` object and have some kind of default function that reads from there.